site/content/integrate/getting-started/integration-contribution-guidelines.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Sharing My Integration"
+date: 2017-09-10T12:33:36-04:00
+subsection: Getting Started
+---
+
+# Sharing my Integration
+
+Contributing an integration is a great way to add new functionality to your system and to share with the community. We want the process to be easy and want to help you produce a great integration that others will enjoy as well!  
+
+Please use the guidelines below to help you get started: 
+
+### Plan your integration
+
+  - Outline the user stories that this integration will support.
+  - Choose what [integration type](https://developers.mattermost.com/integrate/getting-started/how-should-i-integrate/) is ideal for your user stories.
+  - If required by your integration, determine the authentication method best for your integration. (See more information on [Personal Access Tokens](https://docs.mattermost.com/developer/personal-access-tokens.html/) and [OAuth 2.0 applications](https://docs.mattermost.com/developer/oauth-2-0-applications.html/) to help you decide.)
+  - Consider the user experience. Review our [guidelines for UX](https://docs.mattermost.com/developer/fx-guidelines.html) for user experience design for ideas and guidance. We are happy to give feedback as well on our contributors server!
```

Comment:
>our contributors server

We generally call it "our community Mattermost instance" also a link to it would be a good idea